wger/config/models.py

```diff
@@ -148,9 +148,12 @@ def save(self, *args, **kwargs):
                     gym=self.default_gym):
                 user = profile.user
                 if not is_any_gym_admin(user):
-                    try:
-                        user.gymuserconfig
-                    except GymUserConfig.DoesNotExist:
+
+                    # trying to get gymuserconfig of a user no longer
```

Comment:
why did you lose the try catch?

Reply:
Initially the relationship with GymUserConfig was a OneToOne relationship(which made one user belong to one gym), this meant that when we did user.gymuserconfig an exception would be thrown and the except logic would be executed. The relationship was changed to a ForeignKeyField, this does not throw an exception and we have to check that it does not exist manually. I've done this by confirming the length of the set.